src/shared/config/Config.php

```diff
@@ -38,6 +38,13 @@ public function getHomeDirectory(): Directory {
             return $dir;
         }
 
+        if(false !== getenv('PHIVE_HOME')) {
```

Comment:
Yoda should stay with Star Wars ;)

Reply:
The use of `getenv` directly relies on global state. 

Please move that into the `Environment` class.

Reply:
To be clear `PharIo\Phive\Environment::getVariable()` should be used as accessor to environment variables?

Reply:
No, sorry for not being clear and good that you ask :+1: 

If I'm not mistaken, the `$_SERVER` array should already contain all environment variables. Thus, `Environment` should only require additional methods, e.g. something like `Environment::getPhiveHome` and `Environment::hasPhiveHomeVariable` (actual names subject to discussion ;) )

Reply:
Maybe to make it even more clear: The `getVariable` method is generic and only exists in case we need some component to query for a variable we don't know about up front and thus cannot write an explicit getter for.

Reply:
> I'm not mistaken, the $_SERVER array should already contain all environment variables.

That is what I was not sure about but you're right:
```
$ PHIVE_HOME="/whatever" php -r "var_dump(\$_SERVER);" | grep -A1 PHIVE
  'PHIVE_HOME' =>
  string(9) "/whatever"
```

Now everything is clear, I'll update the PR